stages/config/lead/variables.tf

```diff
@@ -26,6 +28,10 @@ variable "harbor_hostname" {
   description = "Harbor hostname provided by app stage"
 }
 
+variable "artifactory_jcr_hostname" {
+  description = "Artifactory hostname provided by app stage"
```

Comment:
let's add a default here too so we don't have to provide this if artifactory isn't enabled